allennlp/semparse/contexts/table_question_knowledge_graph.py

```diff
@@ -88,6 +169,10 @@ def __init__(self,
                  question_tokens: List[Token]) -> None:
         super().__init__(entities, neighbors, entity_text)
         self.question_tokens = question_tokens
+        entity_extractor = QuestionEntityExtractor( '/u/murtyjay/allennlp_fork/stop_words.json',  entity_text)
```

Comment:
Instead of adding this as a field on `TableQuestionKnowledgeGraph`, make this its own class, and test the class directly.  You said you tried this out on some examples and saw that it matched Chen's code - I'd like to see those cases added as test cases for this class in this PR.